README.md

```diff
@@ -56,6 +59,34 @@ pip to install directly from the master branch on github:
 ```
 pip install git+https://github.com/keras-team/keras-nlp.git --upgrade
 ```
+## Using KerasNLP with Keras Core
+
+As of version `0.6.0`, KerasNLP supports multiple backends with Keras Core out 
+of the box. There are two ways to configure KerasNLP to run with multi-backend 
+support:
+
+1. Via the `KERAS_BACKEND` environment variable. If set, then KerasNLP will be 
+using Keras Core with the backend specified (e.g., `KERAS_BACKEND=jax`).
+2. Via the `.keras/keras.json` and `.keras/keras_nlp.json` config files (which 
+are automatically created the first time you import KerasNLP):
+  - Set your backend of choice in `.keras/keras.json`; e.g., `"backend": "jax"`. 
```

Comment:
I played around with this in the markdown preview feature in github. You need at least three spaces before each line item here before this will render as bullets within the numbered list.

Reply:
Nice!